### PR TITLE
TO-DO list progression

### PR DESCRIPTION
**Pull request by KozmikNano**
Completing my to-do list, don't push

### CODE REVIEW

**Comment by KozmikNano** [comment]
I also removed `is2b2t` config option

**Comment by KozmikNano** [comment]
I plan on adding a config version system, that way people can have updated default.json files. (you will see why its needed later).  But this will do for what I need!

**Comment by nickcat325** [comment]
Maybe it would be easier if I could get access to the draft. We should do things together. I like the changes. When are we going to update? I could merge our updates?

File: main.js [hunk 1]
```diff
@@ -188,7 +193,7 @@ function cmdInput() {
 // function to disconnect from the server
 function stop() {
 	webserver.isInQueue = false;
-	finishedQueue = !config.minecraftserver.is2b2t;
+	finishedQueue = false
```
[the remark this answers is not in the document]
Comment (KozmikNano) [reply main.js:196]: No, I removed it becuase it’s useless

[the remark this answers is not in the document]
Comment (nickcat325) [reply main.js:196]: Eh, It's fine if it's hard coded.

[the remark this answers is not in the document]
Comment (KozmikNano) [reply main.js:196]: Yeah but like, we are trying minimize the unneeded trash code

**Comment by nickcat325** [comment]
For some reason it seams to undo my fix. I have no idea why.

File: main.js [hunk 2]
```diff
@@ -254,6 +260,7 @@ function join() {
...
...AcqDSPsNtIlepdjFo/wIAHZz6+vMemcGWHTEhqZp40+W4DJwQB6Qfq1wcyGCIFYANjeFaR8dzKgWWvrcXaXGStP4jso0ONAIgEhb2NGEHTIgiIORc2QiEENf2AaanMNzkIgYo8BUoIJGm/EADmwBKW98vwLjpHU1wTAo/frcykX9sURghIUn4UgCkCIASO3611o8dOPyCNDCO+c5tFuS0xS0AjA9sGpmwHSVuy8eSrXV1dxS6mDrS4zVMY1AiASGD0tsJP7fvtgxBw6wdy+fIRAvcOzjch8KwfUOXrtmKxCelWlH4gdyOymcnRHdYJqgyEuHqomv5N19q6i6bk88//R8bX2GyL77ttTH74MKHXpgWAjIJknBVvLPiJPBl3laygvbfBREHVhKBu/TOAIgPAaO4h01j+RCME1s+IesCoh5HUrTClV5Koua2nrRkBTaAkImoEQCiwt2k+Fn39UbAKgTkXNkhla+pnAmrxGrJUEwJvDs7cTJUhc1uxlF0/YDTQPm5XlXATvrCyIteB9wIqLHlbfCyle6UhhaaJXRICzeMk42usti0amr1bdFMlzNIYyyJZXFDYRzRF/bEYebf/aIr+8MxHVp88cOO/7T8kmAM1AiAKbqZFIGVFaYd1CNzow4Nnzt6jAyZGDoTAPDqgytdVKyNva0IINNGB4nl4yRO4CYGG4pDmlpoktLtXgJ2gUNuwcowO7EfhfwBWXQCkew/+f3t3k5NADAUAuJ3BKCzceAXuYuJF8BQex8u48gocgIWixgXUDZqZMlNphISY71tOH00L74VH+attCCpysebzAyml1+lk+vhemPlcXF5sPj5DO1wyf2ievouleDqwC6tu5GN4Wt/OPflnNADn4ipMwibcj47nhdV0hnoDeTVUnl6N1uv+IWo7FN8OraH0TZiRl+57a+rENSEc41Quf7ugKcSGbuzPBNm+ur8+e8i+BuOyfeX3b2GRxTzYhrP739NTiik+pBiv+1cH8jBvCMIveZDnYnuc0+EmxeXq7ublKJOd2Go9f5vNlovU9q+3g9GH1sEutpfE28KDUTdvTOG54gYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADw330Bletj8G6CdnsAAAAASUVORK5CYII=`)
```
Comment (nickcat325) [inline main.js:263]: This should be `server.favicon = (config.has("favicon") ? config.get("favicon") : null)`

Comment (KozmikNano) [reply main.js:263]: We would like a default favicon. So I disagree respectfully.

**Comment by KozmikNano** [comment]
> For some reason it seams to undo my fix. I have no idea why.

?

**Comment by nickcat325** [comment]
> ?
I'll look into it.

**Comment by KozmikNano** [comment]
> ?
> I'll look into it.

What’s the fix?

**Comment by nickcat325** [comment]
Queue gets stuck at 1.

**Comment by KozmikNano** [comment]
> Queue gets stuck at 1.

Did you fix that?

**Comment by nickcat325** [comment]
Yes but I tried your code and it seems to be broken again. Like I said, i'm looking into it.

**Comment by KozmikNano** [comment]
> Yes but I tried your code and it seems to be broken again. Like I said, i'm looking into it.

No wonder, my code doesn’t have your changes yet

**Comment by KozmikNano** [comment]
When I merge it will

**Comment by rob9315** [comment]
i think i don't have this problem with 2b2wts, i think i solved it by transitioning to playing state once map_chunk packets are received, this changed recently as the server sends a single empty map_chunk packet when first connecting

**Comment by nickcat325** [comment]
Tried it again and it worked correctly. Maybe the queue is buggy.

**Comment by KozmikNano** [comment]
Well, i think its time i merge

**Pull request merged**